authors/apps/authentication/templates/email_verification.html

```diff
@@ -0,0 +1,88 @@
+{% load inlinecss %}
+
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; box-sizing: border-box; font-size: 14px; margin: 0;">
+<head>
+<meta name="viewport" content="width=device-width" />
+<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+<title></title>
+
+
+<style type="text/css">
+img {
```

Comment:
I'm of the opinion that the css in this file should be separate from the html doc. It looks very cluttered making this file unreadable. It would also make the code easier to maintain and manage

Reply:
Also is this the email_verification template? Could you kindly bring me up to speed on what this file should do?

Reply:
With regards to the inline CSS. I did some research and found out 

> I'm of the opinion that the css in this file should be separate from the html doc. It looks very cluttered making this file unreadable. It would also make the code easier to maintain and manage

With regards to the inline CSS I did some research and found out that email clients do not load external CSS that's why I opted for inline CSS. here is the link for the research evidence https://stackoverflow.com/questions/19970348/django-html-e-mail-template-doesnt-load-css-in-the-e-mail

Reply:
> Also is this the email_verification template? Could you kindly bring me up to speed on what this file should do?

The email_verification template is an HTML template that contains the contents of the email to be sent to the user. I opted to use an HTML template to make the email more presentable. Below is a preview of how the email looks like.

<img width="742" alt="Screenshot 2019-04-04 at 20 23 07" src="https://user-images.githubusercontent.com/22986772/55575464-90fee900-5717-11e9-8821-e16f7439a547.png">